### PR TITLE
Speed up

### PR DESCRIPTION
**Pull request by w1158mxs**
Fixed issue #37

- Add the speed-up function and its tests
- Organized code by moving global variables inside the Window class
- Organized cloud tests by moving functions inside their own class

### CODE REVIEW

**Review by Lurk390 — APPROVED** [review]
Putting the global variables and the tests within their own class is a big improvement to the code. Make sure to continue to make the variables uppercase if they are constants or lowercase if they are modified.

**Review by criticalglitch — APPROVED** [review]
Everything looks good, the code is well formatted and the variables are now instanced.

**Review by bakursait2 — APPROVED** [review]
Tests file is well formatted and organized. Putting all related tests in the same class is a good point to start from.

**Review by xieWright — CHANGES_REQUESTED** [review]
In regards to the variable names, the variable d should be distance. Other thing is that I think another issue is popping up in regards to global variables. Gonna need to discuss this with the team.

File: tests/test_minecraft.py [hunk 1]
```diff
@@ -1,37 +1,34 @@
...
-
-def test_non_overlapping_clouds():
-    model = Model()
-    model.generate_clouds(80, 100)
-    blocks_of_all_clouds = [coordinates for coordinates , block in model.world.items() if block == LIGHT_CLOUD]
-    unique_clouds = set(blocks_of_all_clouds)
-    assert len(blocks_of_all_clouds) == len(unique_clouds)
\ No newline at end of file
+from tempus_fugit_minecraft.main import Model
+from tempus_fugit_minecraft.main import LIGHT_CLOUD
```
Comment (Lurk390) [inline tests/test_minecraft.py]: combine these lines together

**Comment by w1158mxs** [comment]
> In regards to the variable names, the variable d should be distance. Other thing is that I think another issue is popping up in regards to global variables. Gonna need to discuss this with the team.

Can you mention which line has variable d, please? I didn't use variable d in my code

**Review by xieWright — APPROVED** [review]

**Comment by w1158mxs** [comment]
contain speed up and organize variables in class in main, and class cloud test

**Comment by w1158mxs** [comment]
Test pass
speed_up_test_screenshot

**Pull request merged**